docs/src/pages/index.page.tsx

```diff
@@ -117,11 +165,15 @@ const HomePage = ({ colorMode }) => {
         backgroundColor={`${tokens.colors.background.secondary}`}
         padding={`${tokens.space.xl}`}
       >
-        <Card style={{ width: '100%', padding: 0 }}>
-          {/* <Sandpack
+        <Heading level={2} textAlign="center" margin={`${tokens.space.xl}`}>
+          Take it for a test drive
+        </Heading>
+        <Card style={{ width: '100%', padding: 0 }} variation="outlined">
+          <Sandpack
             template="react"
             files={{
-              '/App.js': code,
+              '/App.js': { code: code, active: true },
+              '/theme.js': { code: themeCode },
```

Comment:
Heck yeah!! It's back!!!

Reply:
It's ok to have it in, right?

Reply:
Oh, heck yeah!  I just haven't had the time to revisit it. 

Originally, I had React, Angular, & Vue templates and was feeding it code _directly from our proof-of-concept_.

(This was before we had environments, e2e tests, primitives, etc.)

So let's keep it in! This is what we want long-term.